tslib/src/functions/school-pin-http-service/index.ts

```diff
@@ -0,0 +1,21 @@
+
+import { AzureFunction, Context, HttpRequest } from '@azure/functions'
+import { SchoolPinReplenishmnentService } from '../school-pin-generator/school-pin-replenishment.service'
+
+const schoolPinHttpService: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
+
+  const schoolUuid = req.body.school_uuid
+
+  if (!schoolUuid) {
+    context.res = {
+      status: 400,
+      body: 'school_uuid is required'
+    }
+    context.done()
+  }
+  const schoolPinReplenishmentService = new SchoolPinReplenishmnentService()
+  await schoolPinReplenishmentService.process(context.log, schoolUuid)
+  context.done()

```

Comment:
I think if we are using `async`  and `await` then it's probably best practise *not* to call context.done().  I seem to remember reading this, but don't have a link to hand.

Reply:
good catch, as per docs v2 async functions dont need the call...
https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-node#exporting-an-async-function